app/app/src/main/java/com/globant/calculator/android/mvp/view/CalculatorView.java

```diff
@@ -1,6 +1,33 @@
 package com.globant.calculator.android.mvp.view;
 
+import android.app.Activity;
+import android.widget.TextView;
+import com.example.myapplication.R;
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
 public class CalculatorView extends ActivityView {
 
+    @BindView(R.id.calculation_label)
+    TextView calculationLabel;
+    @BindView(R.id.input_label)
+    TextView inputLabel;
+
+    public CalculatorView(Activity activity) {
+        super(activity);
+        ButterKnife.bind(this, activity);
+    }
+
+    public void showResult(Integer result) {
```

Comment:
are you using this somewhere?

Reply:
it's used on calculatorPresenter, onClearButtonPress